HeroVault/HeroVault/Features/User/UserFavouritesScreen/UserFavouritesScreenViewModel.swift

```diff
@@ -52,6 +53,11 @@ class UserFavouritesScreenViewModel: ViewModelDelegate {
         return imageName.convertToHttps()
     }
     
+    func createLabel(marvelDataIndex: Int) -> String {
```

Comment:
this function says create Label but it is just returning a name?
maybe name this a bit better.
maybe fetch Name or something like that?